core/unit_test/IncrementalTest.cpp.in

```diff
@@ -39,18 +39,19 @@
 //
 // ************************************************************************
 //@HEADER
-*/
 
 #ifndef KOKKOS_TEST_@BACK_END_NAME@_@CURRENT_TEST_NAME@
 #define KOKKOS_TEST_@BACK_END_NAME@_@CURRENT_TEST_NAME@
 
 #include <gtest/gtest.h>
+#include <Kokkos_Macros.hpp>
 
-#define TEST_CATEGORY @BACK_END_NAME@
-//#define TEST_EXECSPACE Kokkos::Experimental::@BACK_END_NAME@
-#define TEST_EXECSPACE Kokkos::@BACK_END_NAME@
+#define TEST_CATEGORY @DEVICE@
+#define TEST_EXECSPACE Kokkos::@DEVICE_NAME@
 
+#if defined(KOKKOS_ENABLE_CXX11_DISPATCH_LAMBDA)
```

Comment:
Make sure we only instantiate these guys if we have lambdas available ...

Reply:
:/  not 100% sure it is the right place to do it though.